test/lib/mocked-device-api.ts

```diff
@@ -81,9 +81,15 @@ async function create(): Promise<SupervisorAPI> {
 	// Create SupervisorAPI
 	const api = new SupervisorAPI({
 		eventTracker,
-		routers: [buildRoutes(appManager)],
+		routers: [deviceState.router, buildRoutes(appManager)],
 		healthchecks: [deviceState.healthcheck, apiBinder.healthcheck],
 	});
+
+	const conf = await config.getMany(['macAddress']);
+	deviceState.reportCurrentState({
+		mac_address: conf.macAddress,
+	});
```

Comment:
without this, the device state isn't aware of the MAC address and will not report it...